pyproject.toml

```diff
@@ -8,6 +8,7 @@ license = "BSD 3-Clause \"New\" or \"Revised\" License"
 [tool.poetry.dependencies]
 python = "^3.6"
 SQLAlchemy = "^1.3.20"
+Django = ">=2.2,<4"
```

Comment:
I wouldn't declare `<4`; To me, that's the responsibility of the layer installing aljemy to take care of this.
what if aldjemy is already compatible with 4? that's a pity, if we would need to make a release just to unlock it. given we are testing against the main branch of django, we will probably be ready by the time django4 will be released.

Reply:
Should we adjust other dependences similarly, then? Particularly `SQLAlchemy`, where we are using `^1.3.20`, which is equivalent to `>=1.3.20,<2`.

Reply:
I'm not sure, but I observed in other projects that dependabot willstill attempt to bump to the next major version with `^` (by editing pyptoject.toml file)  and not with the other form.

Reply:
That would be surprising to me. Dependabot does allow you to tell it that it doesn't want to be notified for given ranges of updates. If this doesn't have dependabot doing auto-upgrades, that would be good reason to not use this form, but it would surprise me. I've seen dependabot *add* this form when it was opening things up to a new version of Django.

Reply:
Thanks for sharing, I'll keep an eye on this :)